### ISSUE TITLE
[Bug]: Lost Connection when use Table.col format in query

### ISSUE THREAD

**Issue by SilvaXiang** [matrixorigin/matrixone#1700]
### Is there an existing issue for the same bug?

- [x] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.1.0
- Hardware parameters:docker
- OS type:docker
- Others:docker version
```

### Actual Behavior

<img width="481" alt="image" src="https://user-images.githubusercontent.com/22973642/153640181-7d63a7e0-fb5d-426f-afca-9d23b760a4d2.png">
Result
ERROR 2013 (HY000): Lost connection to MySQL server during query

### Expected Behavior

0

### Steps to Reproduce

```markdown
CREATE TABLE t0(c0 INT);
select c0 from t0;
select t0.c0 from t0;
INSERT INTO t0(c0) VALUES (0);
SELECT t0.c0 FROM t0;
```

**Comment by m-schen** [comment]
Is this issue the same to #1428 ? @nnsgmsone

**Comment by m-schen** [comment]
we try to found t0.c0 while executing output operator, but there is c0 in batch. So we couldn't find it and just return nil to frontend, this will cause the panic.

if we should store t0.c0 in batch ? or vector should record its table name, like {name: c0, table: t0}
and if the sql is
`create table t0 (c0 int, c1 int);`
`create table t1 (c0 int);`
`select c1, t0.c0, t1.c0 from t0 join t1 on t0.c0 = t1.c0;`
what vector's name should we store in the batch?

**Comment by m-schen** [comment]
need a complete rename logic.

**Comment by nnsgmsone** [comment]
fixed by pr #2058

**Comment by domingozhang** [comment]
result is correct
